### PR TITLE
Vhost parameter added.

### PR DESCRIPTION
**Pull request by sergiorb**
I've added the vhost parameter so you can customize it. I'll appreciate any feedback on this PR. Thanks in advance.

### CODE REVIEW

**Review by AlvaroVega — APPROVED** [review]
LGTM

File: docs/installationguide.md [hunk 1]
```diff
@@ -239,6 +240,7 @@ The ones relating specific Ultralight 2.0 bindings are described in the followin
 | IOTA_MQTT_PROTOCOL            | mqtt.protocol           |
 | IOTA_MQTT_HOST                | mqtt.host               |
 | IOTA_MQTT_PORT                | mqtt.port               |
+| IOTA_MQTT_VHOST               | mqtt.vhost              |
```
Comment (fgalan) [inline docs/installationguide.md:243]: As far as I understand, the new vhost parameter is for AMQP configuration, not MQTT (looking to "twin" PR https://github.com/telefonicaid/iotagent-json/pull/527 it only appears in the AMQP section of the configuration).

Is this a typo? Or maybe I'm missing something? :)

Comment (sergiorb) [reply docs/installationguide.md]: Hello @fgalan and sorry for the delay. Yes, it was a typo. I've updated the PR. Thanks.

File: package-lock.json [hunk 2]
```diff
@@ -0,0 +1,7941 @@
+{
```
Comment (fgalan) [inline package-lock.json:1]: This file should be removed from PR.

File: .github/workflows/ci.yml [hunk 3]
```diff
@@ -1,5 +1,5 @@
 name: CI
```
Comment (fgalan) [inline .github/workflows/ci.yml:1]: Why this file has been changed in the PR? 

In addition, looking to https://github.com/telefonicaid/iotagent-json/pull/527 I don't see similar changes

**Comment by fgalan** [comment]
There were some changes on that iotagent-node-lib library that make some tests to be "unaligned". It has been already fixed in master (by PR #476 ).

Thus, could you please update this PR brach con master. After that I guess that all your changes in ci.yml in this PR are not needed and can be reverted. Remember also to remove packages-lock.json from the PR. Thanks!

**Comment by fgalan** [comment]
@sromerotech could you explain why this PR has been closed, please? Do you plan to re-create it? Thanks!

**Pull request closed without merging**